bitnami/postgresql/Chart.yaml

```
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 11.6.21
+version: 11.6.22
```

There is a new version (https://github.com/bitnami/charts/pull/11548) so we need to bump it again
```suggestion
version: 11.6.23
```